opencog/relex/Dockerfile

```diff
@@ -0,0 +1,98 @@
+#
+# Docker file that builds RelEx and starts the RelEx server.
+#
```

Comment:
sure. If the buidbot makes a nightly build of relex. Then this could be a just a relex server . Actually the same could be done for the cogserver and moses . Have buildbot make an rpm and/or deb package and have those be installed.